string arm_id 
bool coordinated
---
bool success